variables.tf

```diff
@@ -72,6 +72,12 @@ variable "redis_version" {
   default     = null
 }
 
+variable "redis_configs" {
+  description = "The Redis configuration parameters."
```

Comment:
nit: Can we add a reference to where a user can go to find [additional configs](https://www.terraform.io/docs/providers/google/d/redis_instance.html#redis_configs)? Perhaps similar to [this]( https://github.com/terraform-google-modules/terraform-google-sql-db/blob/76244bdeed4896dbad645345b11c71df2ddfc87a/modules/mysql/variables.tf#L119)

Reply:
> nit: Can we add a reference to where a user can go to find [additional configs](https://www.terraform.io/docs/providers/google/d/redis_instance.html#redis_configs)? Perhaps similar to [this](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/76244bdeed4896dbad645345b11c71df2ddfc87a/modules/mysql/variables.tf#L119)

Done